analyses/sample-distribution-analysis/02-multilayer-pie.R

```diff
@@ -0,0 +1,91 @@
+# This script creates a treemap and a multilayer pie chart to represent the
+# broad histologies, short histologies, and molecular subtypes within the dataset.
+#
+# This script uses the packages sunburstR, d3r, and treemap to produce the
+# visualizations.
+#
+# Chante Bethell for CCDL 2019
+
+# Load/install packages. This will be removed once a Dockerfile is established.
+source("00-install-packages.R", print.eval = FALSE)
+
+# Load source script
+source("01-filter-across-types.R")
+
+# Set path to plots directory
+outputDir <- file.path("/plots")
+
+# magrittr pipe
+`%>%` <- dplyr::`%>%`
+
+# Create a colorblind-friendly color vector
+color <- colorblindr::palette_OkabeIto
+
+sun_df <- df2 %>%
```

Comment:
Once you remove `source("01-filter-across-types.R`")`, you will need to read this file in.